src/main/java/net/mcreator/blockly/java/blocks/JavaCodeExecuteBlock.java

```diff
@@ -24,15 +24,14 @@
 import net.mcreator.util.XMLUtil;
 import org.w3c.dom.Element;
 
-public class JavaCodeBlock implements IBlockGenerator {
+public class JavaCodeExecuteBlock implements IBlockGenerator {
 
 	@Override public void generateBlock(BlocklyToCode master, Element block) {
 		Element element = XMLUtil.getFirstChildrenWithName(block, "field");
 		if (element != null) {
 			master.append(element.getTextContent()).append("\n");
 		} else {
-			master.addCompileNote(
-					new BlocklyCompileNote(BlocklyCompileNote.Type.WARNING, "Empty code block was skipped."));
+			master.addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.WARNING, "Empty code block was skipped."));

```

Comment:
Why did you refactor lines like this? Please revert all occurrences as to limit the number of changes in a single PR to only the necessary ones that are relevant to the issue being fixed/done.

Reply:
Such changes can also be seen throughout the origin repository.
If @KlemenDEV requests reverting line changes (he is _the main developer_), I'll fix that.

Reply:
> Such changes can also be randomly seen throughout the origin repository.

This is not really an excuse to do more of such changes, though. I need to run code formatter than later and possibly break PRs with merge conflicts.

Reply:
So should I revert this or not?

Reply:
I mean it is generally better to not manually format, but you can leave it. I plan to run code reformat in the future anyway ;)

Reply:
Seems it's practically fine.